### PR TITLE
Initialization: Set up project

### PR DESCRIPTION
**Pull request by MarcusNguyen1608**
**Tasks**

- Setup base project
- Login, Logout, Captcha Method
- Private Route

**Description**

- Update source structure
- Install Eslint + Prettier + Husky & config
- Install react-router-dom V6
- Install redux + redux-toolkit + persist & config
- Install reselect
- Install dotenv & create .env file & .env.example
- Install react-18next
- Install react-hool-form + yup
- Install storybook & create example
- Install react-select
- Install date-fns
- Install cypress
- Install jest
- Install uuid
- Install react-toastify
- Handle Private Route & Login & Logout, Forgot password method
- Install classnames library
- Add icons & images

**Screenshots**

image
image
image

### CODE REVIEW

File: src/wdyr.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (leo-nguyen-goldenowl) [inline src/wdyr.js:1]: move `wdyr` to `services` folder

Comment (ghost) [inline src/wdyr.js]: move `wdyr` to `services` folder

File: src/views/PageNotFound/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import Loader from 'components/Loader';
```
Comment (ghost) [inline src/views/PageNotFound/index.js]: Create `pages` in `views`
```
views
|__ pages
|_____ NotFound
|_____ SignIn
|_____ SignUp
|__ Cards
|__ Contacts
|__ Transfer
|__ Debts
|__ History
|__ Account
    
```

**Review by ghost — CHANGES_REQUESTED** [review]
Do not capitalize the names of images, icons
Please use camel case and define the meaning names, like avoid `Alert-1`???

Do not create folder levels in images and icons
Notification > Busy, Free, Remove => notificationBusy, notificationFree, notificationRemove
If using `SVG`, should not use img source, we should use Svg components

Remove .gitkeep if dont need

@MarcusNguyen1608

**Comment by ghost** [comment]
Please update then remember to tell me know @MarcusNguyen1608

**Pull request merged**